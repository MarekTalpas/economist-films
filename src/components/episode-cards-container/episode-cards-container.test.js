// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import EpisodeCardsContainer from './episode-cards-container';

test('EpisodeCardsContainer renders correctly', () => {
  const tree : string = renderer.create(<EpisodeCardsContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
