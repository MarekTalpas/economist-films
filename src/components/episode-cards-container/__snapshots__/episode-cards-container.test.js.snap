exports[`test EpisodeCardsContainer renders correctly 1`] = `
<div
  className="episode-cards">
  <h1
    className="episode-cards__title">
    Recommended
  </h1>
  <div
    className="episode-cards__wrapper" />
</div>
`;
